src/scss/index.scss

```diff
@@ -1,18 +1,21 @@
 @import './normalize.scss';
-@import './components/App/App';
-@import './components/Header/Header';
-@import './components/Main/Main';
-@import './components/Score/Score';
+@import './variables.scss';
```

Comment:
I also made a variables file, but its in the folder variables and named index.js its NOT for CSS variables, so you dont have to worry about merging them with this doc